scripts/test.sh

```diff
@@ -20,13 +20,13 @@ fi
 
 case $PLATFORM in
   MACOS)
-      echo "Building for Apple Darwin x86_64"
-      rustup target add x86_64-apple-darwin
+      echo "Building for Apple Darwin aarch64"
+      rustup target add aarch64-apple-darwin
       case $LANGUAGE in
         C)
           echo "To be used with C"
-          cargo build --target x86_64-apple-darwin --release
-          export RUST_LIBRARY_DIRECTORY="${PWD}/target/x86_64-apple-darwin/release"
+          cargo build --target aarch64-apple-darwin --release
+          export RUST_LIBRARY_DIRECTORY="${PWD}/target/aarch64-apple-darwin/release"
```

Comment:
Just wondering if you think we should make the `--target` an environment variable? And have the CI pass it in?

Reply:
Just keep in mind we are currently releasing using the `x86_64-apple-darwin` target to release. Only for the `MACOS` target so if this doesn't work then it won't work on the release.

We might need both

https://github.com/mattrglobal/ffi-bbs-signatures/blob/ee979624ac1bff78adaac60bd6b0ec546cc73413/scripts/build.sh#L51-L56

Reply:
That said on the seems like we are only releasing `IOS` and `Android` 

https://github.com/mattrglobal/ffi-bbs-signatures/blob/ee979624ac1bff78adaac60bd6b0ec546cc73413/.github/workflows/release.yml#L24

https://github.com/mattrglobal/ffi-bbs-signatures/blob/ee979624ac1bff78adaac60bd6b0ec546cc73413/package.json#L11-L14

Note `yarn build` does not include the macos job. So it's probably fine just something to note.